model/atmosphere/dycore/src/icon4py/model/atmosphere/dycore/velocity/velocity_advection.py

```diff
@@ -73,32 +76,84 @@ def __init__(
 
         self._initialized = True
 
+        self.mo_icon_interpolation_scalar_cells2verts_scalar_ri_dsl = (
```

Comment:
As for `SolveNonHydro` can you mark them all private by prefixing with `_`:
```suggestion
        self._mo_icon_interpolation_scalar_cells2verts_scalar_ri_dsl = (
```